assignments/assignment7/style/style.scss

```diff
@@ -0,0 +1,375 @@
+$graycolor:darkslategray;
+$tealcolor:teal;
+$font-style:sans-serif,italic;
+
+body,
+.header,
+.main-wrap {
+    margin: 0px auto;
+}
+
+#headerleft,
+#leftcontent {
+    box-sizing: border-box;
+}
+
+.header,
+.maindot,
+#leftcontent,
+#rightcontent {
+    overflow: hidden;
+}
+
+#headerleft h1,
+address,
+#english-level,
+#arabic-level,
+.h4-language,
+.ul-skills li p,
+.start-time,
+.end-time,
+.place {
+    display:inline;
```

Comment:
I am not sure if this is readable as compared to moving the `display:inline;` to the respective classes in .place, .end-time etc.